### ISSUE TITLE
Fix broken unit tests

### ISSUE THREAD

**Issue by PlagueHO** [PowerShell/xStorage#76]
I'm not sure why, but a number of the unit tests have broken since the last release:

https://ci.appveyor.com/project/PlagueHO/xstorage/build/2.9.59.0

It appears that a recent pester (4.0.2) change has broken these. Pester version 3.4.6 works correctly. It appears to be a pester bug that prevents Get-Volume from being mocked in two successive Context blocks. Only the first Mock works. The second Mock does not mock the call and so the underlying cmdlet is called instead.

**Comment by PlagueHO** [comment]
I believe this may be caused by an issue in Pester and have logged it:
https://github.com/pester/Pester/issues/697

**Comment by PlagueHO** [comment]
I've submitted a change to force the old version of pester to be used in AppVeyor. I'll leave this issue open until Pester 4.0.2 bugs are fixed. Once that is done I'll submit another PR to allow the new version of Pester to be used.
